qualtran/bloqs/bookkeeping/join.py

```diff
@@ -95,7 +94,7 @@ def my_tensors(
         ]
 
     def on_classical_vals(self, reg: 'NDArray[np.uint]') -> Dict[str, int]:
-        return {'reg': bits_to_ints(reg)[0]}
+        return {'reg': self.dtype.from_bits(reg.tolist())}
```

Comment:
why is `tolist` required?

Reply:
mypy

Reply:
it doesn't recognize `ndarray` as `Sequence`?

Reply:
expects Sequence[int] but got ndarray[np.int64]